gremlin-server/src/test/java/org/apache/tinkerpop/gremlin/server/GremlinDriverIntegrateTest.java

```diff
@@ -71,7 +73,7 @@
  * @author Stephen Mallette (http://stephen.genoprime.com)
  */
 public class GremlinDriverIntegrateTest extends AbstractGremlinServerIntegrationTest {
-
+    private static final Logger logger = LoggerFactory.getLogger(AbstractGremlinServerIntegrationTest.class);
```

Comment:
This should be `LoggerFactory.getLogger(GremlinDriverIntegrateTest .class)`, right?